librad/src/git/storage.rs

```diff
@@ -747,6 +747,8 @@ where
             .copied()
             .collect::<HashSet<PeerId>>();
 
+        // FIXME(haxpenny): wtf is a resolver... we need to verify the certifier
+        // delegations that are a retrieved from this fetch.
```

Comment:
Do we not want to ensure `refs/namespaces/<certifier>/rad/{id,ids/*}` exists, even though we didn't actually verify?

Reply:
Yup, strong point well made :metal:

Reply:
Although, wait a minute. It's not `refs/namespaces/<certifier>/rad/{id,ids/*} ` but instead `refs/namespaces/<certifier>/refs/remotes/<tracked_peer>/rad/{id,ids/*} ` because the refspec in `fetch_heads` doesn't map it to a top level.

Which I suppose begs the question, does it make sense to add a fetch so that we have `refs/namespaces/<certifier>/rad/{id,ids/*}`? I guess I don't see why not.

Reply:
Actually, came to see a reason why not. We would have to come to a decision on which is the "canonical" version if we have multiple peers under the same certifier. So we probably shouldn't make this arbitrary choice and instead just have them under each remote.

And so the validation for existence is for `refs/namespaces/<certifier>/refs/remotes/<tracked_peer>/rad/{id,ids/*}`

Reply:
My mental model would match a layout similar to how projects work as well where potentially different versions of identity materialise under a remote of a tracked peer. When viewing such an identity, i.e. another users profile in app, one would select which peers view to look at similar again to how projects are interacted with.

Reply:
Yes. It may at this point that we either _already have_ `refs/namespaces/<xxx>/rad/id` (because we cloned that independently), or that we don't. In the former case, what the code currently does is correct (merging updates is an interactive process), whereas the latter would require proper verification. Since we don't yet have that, we can just pretend for now that the remote view is fine, and adopt this as our own view.

Reply:
So if an "Entity" is signed by `Signatory::OwnedKey` it doesn't have any `rad/ids/*` :facepalm: 

So my thinking to fulfil this is: I can verify that `refs/namespaces/<certifier>/refs/remotes/<remote_peer>/rad/id` exists and adopt that as our own. Commit incoming.

Reply:
> So my thinking to fulfil this is: I can verify that `refs/namespaces/<certifier>/refs/remotes/<remote_peer>/rad/id` exists and adopt that as our own. Commit incoming.

What is meant by adopting as our own here?

Reply:
> So if an "Entity" is signed by `Signatory::OwnedKey` it doesn't have any `rad/ids/*`

Should that be addressed?

Reply:
> What is meant by adopting as our own here?

We create a `symref` of `refs/namespaces/<certifier>/refs/remotes/<remote_peer>/rad/id` under `refs/namespaces/<certifier>/refs/rad/id` where `remote_peer` is the peer we're cloning from. This is the same behaviour as with projects.

> Should that be addressed?

I'm not sure it's worth it because Entities won't be around much longer and I don't think this necessarily breaks things.